.github/workflows/publish.yml

```diff
@@ -27,5 +27,5 @@ jobs:
         TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
         TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
       run: |
-        python setup.py sdist bdist_wheel
+        python -m setup.py sdist bdist_wheel
```

Comment:
Doesn't this try to load a module named `setup.py`?

Reply:
I suppose? 

I just tried running `python setup.py sdist bdist_wheel` without the `-m` flag and it wouldn't work, and then when I did it with the `-m` flag (as I've seen in other repositories) it worked as expected.

Reply:
With the `-m` flag I get:
```
python: Error while finding module specification for 'setup.py' (ModuleNotFoundError: __path__ attribute not found on 'setup' while trying to find 'setup.py')
```

This is on OSX with Python 3.7.6

Reply:
Did you do that after `python3 -m venv ./venv && source venv/bin/activate`?

Reply:
Yes, from within the virtualenv.

Reply:
FWIW, this one came from Github's own [starter workflows](https://github.com/actions/starter-workflows/blob/main/ci/python-publish.yml#L30). As such, I'd anticipate it working with Github Actions so I'd probably err away from modifying it.

Reply:
Will do. Apologies; I'll fix that shortly

Reply:
Fixed in most recent commit